src/pages/notice/NoticeList.tsx

```diff
@@ -1,3 +1,204 @@
+      sort: sortMap[sort] ?? undefined,
+    })
+      .then((data) => {
+        setAllNotices(data.items.map((item) => item.item));
+        setTotalCount(data.count);
+      })
+      .catch((err) => setError(err.message))
+      .finally(() => setLoading(false));
+  }, [search, sort, filterValues, currentPage]);
+
+  const totalPages = Math.ceil(totalCount / ITEMS_PER_PAGE); // 페이지네이션
+  const appliedFilterCount = countAppliedFilters(filterValues); // 필터 적용
+
+  // 필터 적용
+  const handleApplyFilter = (values: FilterValues) => {
+    setFilterValues(values);
+    setCurrentPage(1);
+  };
+
+  // ===== 렌더 =====
+  if (loading)
+    return (
+      <div className="flex h-500 items-center justify-center text-h3 text-black">
+        잠시만 기다려주세요
+      </div>
+    );
+  if (error)
+    return (
+      <div className="flex h-500 items-center justify-center text-h3 text-red-500">
+        {error}
+      </div>
+    );
+
+  return (
+    <main>
+      {/* 맞춤 공고 */}
+      {!search && (
+        <article className="bg-red-10 px-32 py-60">
+          <section className="mx-auto flex max-w-964 flex-col">
+            <h2 className="mb-32 text-h1 font-bold">맞춤 공고</h2>
+            <div className="flex gap-14 overflow-hidden">
+              {allNotices.slice(0, 3).map((notice) => (
+                <Link
+                  key={notice.id}
+                  to={`/shops/${notice.shop.item.id}/notices/${notice.id}`}
+                  className="block"
+                >
+                  <Post data={notice} />
+                </Link>
+              ))}
+            </div>
+          </section>
+        </article>
+      )}
+
+      {/* 전체 공고 */}
+      <article className="px-32 py-60">
+        <section className="mx-auto flex max-w-full flex-col gap-32 sm:max-w-964">
+          <div className="flex flex-col gap-12 sm:flex-row sm:justify-between">
+            <h2 className="mb-4 text-h1 font-bold sm:mb-0">
+              {search ? (
+                <>
+                  <span className="text-h1 font-bold text-primary">
+                    {search}
+                  </span>
+                  <span className="text-h1 font-bold">에 대한 공고 목록</span>
+                </>
+              ) : (
+                '전체 공고'
+              )}
+            </h2>
+            <div className="flex gap-10">
+              <Dropdown
+                options={SORT_OPTIONS}
+                placeholder="마감임박순"
+                variant="filter"
+                selected={sort}
+                setSelect={setSort}
+              />
+              <div className="relative">
+                <button
+                  className="rounded-md bg-red-30 px-12 py-8 text-body2 text-white"
+                  type="button"
+                  onClick={() => setFilterOpen(true)}
+                >
+                  상세필터
+                  {appliedFilterCount > 0 && (
+                    <span className="ml-2">({appliedFilterCount})</span>
+                  )}
```

Comment:
```suggestion
                  {appliedFilterCount > 0 && (
                    ' ' + appliedFilterCount
                  )}
```
💬 이런 식으로도 가능하지 않을까요?

Reply:
간결하고 더 좋네요 리펙토링때 수정하겠습니다!